src/clj/vanilla/db/core.clj

```diff
@@ -22,6 +22,10 @@
   "SQLite database connection spec."
   {:dbtype "sqlite" :dbname "vanilla_db"})
 
+(def users-db
```

Comment:
don't think we want a second database file. could cause trouble in a deployed docker container, as we have to set up the single Database-URL arg for the app.

Reply:
Agree with having only a single db for the app.
Does Chris's comment about "single Database-URL" mean we'll have problems accessing another system's db?

Reply:
We need to look into how we can specify db config information to the app when it’s inside a deployed container. Some stuff may be “working” simply because thing running local still have access to local resources. 

But we don’t want OUR data for running/configuring Vanilla in multiple places. Reaching to other data sources is different

Reply:
Scratch that. We didn’t use an external config for the database(s). We hard coded them into db.core. 

Let’s talk about the hardcoding, but I think we should still get down to 1 db for “internal” use

Reply:
Yeah that shouldnt be a problem, I think chad just had it already created when we merged together.  Should be quick to just make it a table in vanilla db, will do.

Reply:
Remove 3 lines and change the few calls that use user_db

Reply:
Done and tested

Reply:
Yeah just to comment on it, user_db was created for testing purposes. Thanks for taking care of it @austinpaine !